package.json

```diff
@@ -75,6 +75,7 @@
     "eslint-plugin-rxjs": "^5.0.3",
     "jest": "^29.7.0",
     "lintspaces-cli": "^0.7.1",
+    "markbind-cli": "^5.3.0",
```

Comment:
This shouldn't be needed. Only the `package.json` file in `docs/` folder needs the change.

Reply:
Hi can I clarify the reason? Isn’t it better for all package.json files to have the same latest version of markbind? 

On the same note, can I also clarify if I changed back this package.json file, I’ll also need to change back the package-lock.json?

Reply:
@jingting1412 I believe you have figured it out, but the main project (thus root `package.json`) has nothing to do with MarkBind. This is not a Lerna monorepo project whereby the root `package.json` affects the one in the docs folder.

Reply:
Yes understood thank you, I have since reverted the changes I've made to the main json files.